configurations/ldap-authorization-and-authentication.yaml

```diff
@@ -0,0 +1,8 @@
+authenticator: 'com.scylladb.auth.SaslauthdAuthenticator'
```

Comment:
Let's add a new short longevity (100gb-4h) that use this configuration (authorization + authentication)

Reply:
fixed.